package.json

```diff
@@ -78,7 +78,7 @@
     "eslint": "^8.7.0",
     "expect": "^27.4.6",
     "fast-check": "^2.21.0",
-    "ipfs-core": "^0.13.0",
+    "ipfs-core": "^0.14.2",
```

Comment:
This can be scary, but from looking at the [CHANGELOG](https://github.com/ipfs/js-ipfs/blob/master/packages/ipfs-core/CHANGELOG.md#0140-2022-01-27) it should be fine :+1: